docs/CONTRIBUTING.md

```diff
@@ -68,7 +57,10 @@ $ /usr/bin/safaridriver --enable
     1. A draft release has been created for you. Artifacts are already uploaded.
     1. If you can't find the release matching your desired version, then there might be issues with the Release Action. Navigate to <https://github.com/signalfx/splunk-browser-otel-js/actions>, and investigate the right CD run.
     1. Fill out release title, release notes
-    1. "Publish" the release.
+1. run `npm run release:cdn`
+    1. this script updates release contents and will ask you to verify them (URL will be provided)
+    1. verify that CDN works by running `npm run test:integration:local:chrome`, one of the tests uses
```

Comment:
If there's only one test doing it maybe separate command like `npm run test:integration:verify-cdn` to only run that one test?